src/renderer/extensions/vueNodes/composables/useNodePointerInteractions.test.ts

```diff
@@ -193,8 +234,17 @@ describe('useNodePointerInteractions', () => {
       mockOnNodeSelect
     )
 
-    // Start drag
-    pointerHandlers.onPointerdown(createPointerEvent('pointerdown'))
+    // Pointer down alone shouldn't set dragging state
+    pointerHandlers.onPointerdown(
+      createPointerEvent('pointerdown', { clientX: 100, clientY: 100 })
+    )
+    await nextTick()
```

Comment:
nit: Usually when I see `await nextTick()` in a test, it means that there's an action that could/should have used Vue test-utils (which does the awaiting for you)